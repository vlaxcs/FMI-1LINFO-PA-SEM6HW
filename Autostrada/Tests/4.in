5123
120 312
412 504
512 950
940 1200
4500 210
132 133
1600 1900
4000 4600