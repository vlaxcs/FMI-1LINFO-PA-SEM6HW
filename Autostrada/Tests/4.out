[120, 312]
[412, 504]
[512, 1200]
[1600, 1900]
[4000, 4600]

(0, 120)
(312, 412)
(504, 512)
(1200, 1600)
(1900, 4000)
(4600, 5123)

37%